### PR TITLE
Added integration files for Lever

### PR DESCRIPTION
**Pull request by hassan254-prog**
## Describe your changes
Added sync and action integration templates for Lever. 

## Test
I have successfully implemented synchronization for lever, utilizing this provider to establish connections in local development. The integration has proven effective in syncing Lever opportunities, applications, feedback, interviews, notes and offers for a specific opportunity. Also syncing postings and questions for a specific job posting. The templates includes actions to create a note for a specific opportunity and an opportunity.

### CODE REVIEW

File: docs-v2/integration-templates/overview.mdx [hunk 1]
```diff
@@ -101,6 +101,10 @@ As Nango and its community expand, we're looking forward to offering hundreds of
     Sync Zendesk tickets and articles
   </Card>
 
+  <Card title="Lever" href="/integration-templates/lever" icon="briefcase">
+    Sync Lever opportunities, applications, feedback, interviews, notes and offers for a specific opportunity. Postings and questions for a specific job posting. Actions to create; a note for a specific opportunity and an opportunity.
+  </Card>
+
```
Comment (TBonnin) [inline docs-v2/integration-templates/overview.mdx:107]: I think there is a missing word at the end `and an opportunity`. also extra `;` after `create`

File: integration-templates/lever/lever-create-note.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+        const urlWithParams = `${endpoint}${params ? `?${params}` : ''}`;
+
+        const resp = await nango.post({
+            endpoint: urlWithParams,
+            data: postData
+        });
+
+        return {
+            id: resp.data.data.id,
```
Comment (TBonnin) [inline integration-templates/lever/lever-create-note.ts:51]: is `data.data` expected?

**Review by TBonnin — APPROVED** [review]
small comments but looks good otherwise

File: integration-templates/lever/lever-create-note.ts (hunk 2 above)
Comment (hassan254-prog) [reply integration-templates/lever/lever-create-note.ts:51]: Yes, the response has a nested `data` property within the `data` property. 
notes

**Pull request merged**